client/pages/_app.js

```diff
@@ -0,0 +1,7 @@
+import 'tailwindcss/tailwind.css'
+
+function MyApp({ Component, pageProps }) {
```

Comment:
nit: fat arrow declaration?

Reply:
Oops! Project was not created with typescript version - fixed.